allennlp/data/iterators/bucket_iterator.py

```diff
@@ -88,16 +88,17 @@ def _sort_dataset_by_padding(dataset: Dataset,
         """
         instances_with_lengths = []
         for instance in dataset.instances:
-            padding_lengths = instance.get_padding_lengths()
+            padding_lengths = cast(Dict[str, Dict[str, float]], instance.get_padding_lengths())
```

Comment:
something really dumb. `instance.get_padding_lengths()` returns the lengths as `int`s, but `add_noise_to_dict_values()` wants the values to be `float`s.  so it's basically saying "treat the ints as floats" (`cast` is a no-op at runtime)